druid-shell/src/mac/mod.rs

```diff
@@ -136,7 +136,6 @@ impl WindowBuilder {
             window.autorelease();
             window.cascadeTopLeftFromPoint_(NSPoint::new(20.0, 20.0));
             window.setTitle_(make_nsstring(&self.title));
-            window.makeKeyAndOrderFront_(nil);
```

Comment:
an alternative solution is to move this line down to the end of this function, after we've done all of our view setup.

I'm not totally sure which one is 'better', but both fix the issue of not having mouse when we launch.